sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_directory_client_async.py

```diff
@@ -281,14 +281,14 @@ def get_subdirectory_client(self, directory_name: str, **kwargs) -> Self:
 
         _pipeline = AsyncPipeline(
             transport=AsyncTransportWrapper(self._pipeline._transport),  # pylint: disable=protected-access
-            policies=self._pipeline._impl_policies  # pylint: disable=protected-access
+            policies=self._pipeline._impl_policies  # type: ignore # pylint: disable=protected-access
```

Comment:
Is `# type: ignore` sufficient, or should I do something more specific like `# type: ignore [arg-type]`

Reply:
If it can be more specific then probably prefer to do that in case there is any other violations that crop up later, we know about it and it isn't getting silently swallowed.

Reply:
Sounds good! 😎